lib/stat_tracker.rb

```diff
@@ -340,4 +340,17 @@ def average_win_percentage(team_id)
     wins > 0 ? ((wins.to_f / games)).to_f.round(2) : 0
   end
 
+  def worst_loss(team_id) 
+    worst_loss_margin = []
+
+    @all_games.each do |game|
+      # binding.pry
```

Comment:
remove this please, otherwise code looks good and functions well

Reply:
it has been removed